### PR TITLE
Joke Tank

### PR DESCRIPTION
**Pull request by Hughgent**
## About The Pull Request

It's the tank, but with mouth noises!

## Why It's Good For The Game

Yeah it's a joke.

## Changelog
:cl: Hughgent
add: Admins can now toggle optional funny tank noises.
/:cl:

### CODE REVIEW

**Comment by Hughgent** [comment]
https://youtu.be/zHGcHIa7QEY

I regret nothing I have done here.

add "FUN_ALLOWED" to config.txt for it to work.

File: code/modules/admin/fun_verbs.dm [hunk 1]
```diff
@@ -37,6 +37,26 @@
...
 	message_admins("[ADMIN_TPMONTY(usr)] has gibbed themselves.")
 
+/datum/admins/proc/tank_mouth_noises()
+	set name = "Toggle Tank Mouth Noises"
+	set category = "Fun"
+
+	if(!check_rights(R_FUN))
+		return
+
+	if(istype(usr, /mob/dead/observer))
```
Comment (DominikPanic) [inline code/modules/admin/fun_verbs.dm]: Most admins are observers. This check is unnecessary.

File: code/modules/admin/fun_verbs.dm [hunk 2]
```diff
@@ -37,6 +37,26 @@
...
+		return
+
+	if(alert(usr, "Are you sure you want to make the tank sound like a five year old driving it?", "Warning" , "Yes", "No") != "Yes")
+		return
+
+	if(!usr || istype(usr, /mob/dead/observer))
```
Comment (DominikPanic) [inline code/modules/admin/fun_verbs.dm]: Same here.

File: code/modules/admin/fun_verbs.dm [hunk 3]
```diff
@@ -37,6 +37,26 @@
...
+		return
+
+	GLOB.tank_mouth_noise = !(GLOB.tank_mouth_noise)
```
Comment (DominikPanic) [inline code/modules/admin/fun_verbs.dm]: Make it a config instead of a GLOB.

**Comment by Hughgent** [comment]
Compiles locally and functions flawlessly.

**Pull request merged**